harvard-huit/Harvard-Geospatial-Library
# Resolved bugs related to map & search results page

**Resolved bugs related to map & search results page*
* * *

**JIRA Ticket**: * [HGL-465: Improve "click to wake" button readability](https://jira.huit.harvard.edu/browse/HGL-465)

# What does this Pull Request do?
* Leaflet map was acting buggy (after interacting with it, map would turn gray and any custom js added to search results page would get removed). This PR resolves those issues by removing turbolinks from two js files (`application.js` and `bootstrap-accordion.js`) and updating the `results.js` file (more specifically the updatePage() function on lines 65-80) to prevent the filter rail and icon labels from being replaced on search results page.
* Styling updates were also made to the map so that the "click to wake" button is more readable

# How should this be tested?
Check for the following:
* "Click to wake" button on map has a more opaque background, black border + text, and text is all caps with a larger font
* On the [search results page](https://localhost:3001/), click around on different filters and watch the map recenter itself/zoom in depending on the filter you choose ("subject" and "place" are good categories to test this with...though a heads up that with filters like "Massachusetts" or "Boundaries" the map doesn't really zoom in much/at all because the area for at least one of the items with that filter tag covers basically the whole map)
* Execute a search by zooming in on the map and confirm that (1) you still see labels next to the icons for each search results item and (2) the "expand/collapse all" button still works as intended
* View the metadata for [this item](https://localhost:3001/catalog/stanford-cg357zz0321#iso-identification-info) and confirm that the table of contents anchor links (in the iso tab) work and jump you down the list to the appropriate section

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl 
@enriquediaz
